components/contentblocks/indicator-chart/DashboardIndicatorLineChartBlock.tsx

```diff
@@ -0,0 +1,263 @@
+    });
+  }
+
+  const seriesLines = buildLines(dimSeries);
+  const seriesTotal =
+    showTotalLine && totalRaw.length ? buildLines([totalDef], 3) : [];
+
+  // Trend line
+  const regData = totalRaw.slice(-Math.min(totalRaw.length, 10));
+  const predictedYears = regData.map(([yr]) => yr);
+  if (indicator?.goals?.length) {
+    const highestGoalYear = Math.max(
+      ...indicator.goals.map((g) => new Date(g.date).getFullYear())
+    );
+    if (highestGoalYear > predictedYears[predictedYears.length - 1]) {
+      predictedYears.push(highestGoalYear);
+    }
+  }
+  const model = linearRegression(regData);
+  const predictedValues = predictedYears.map((yr) => model.m * yr + model.b);
+
+  const trendSeries =
+    regData.length >= 2 &&
+    (graphsTheme.showTrendline ?? indicator?.showTrendline)
+      ? [
+          {
+            name: 'Trend',
+            type: 'line' as const,
+            symbol: 'none',
+            showSymbol: false,
+            smooth: false,
+            data: predictedYears.map(
+              (yr, i) => [String(yr), predictedValues[i]] as [string, number]
+            ),
+            lineStyle: {
+              type: 'dashed',
+              width: 2,
+              color: graphsTheme.trendLineColor ?? '#aaa',
+            },
+            itemStyle: {
+              color: graphsTheme.trendLineColor ?? '#aaa',
+            },
+            tooltip: { show: false },
+          },
+        ]
+      : [];
+
+  const goalSeries =
+    indicator?.goals?.map((g) => {
+      const y = new Date(g.date).getFullYear();
+      return {
+        name: 'Goal',
+        type: 'scatter' as const,
+        symbol: X_SYMBOL,
+        symbolSize: 10,
+        data: [[String(y), g.value]],
+        itemStyle: { color: graphsTheme.goalLineColors?.[0] ?? '#3E9C88' },
+        tooltip: { formatter: `Goal: ${g.value} ${unit}` },
+      };
+    }) || [];
+
+  const legendData = [
+    ...dimSeries.map((d) => d.name),
+    ...(seriesTotal.length ? ['Total'] : []),
+    ...(goalSeries.length ? ['Goal'] : []),
+    ...(trendSeries.length ? ['Trend'] : []),
+  ];
+
+  const option: ECOption = {
+    legend: {
+      show: true,
+      data: legendData,
+      left: 'center',
+      bottom: 0,
+      textStyle: { color: theme.textColor.secondary },
+    },
+    tooltip: {
+      trigger: 'axis',
+      appendTo: 'body',
+      axisPointer: { type: 'line' },
+      formatter: (params) => {
+        const seen = new Set<string>();
+        const ary = Array.isArray(params) ? params : [params];
+        const year = ary[0]?.data?.[0];
+        const rows = ary
+          .filter((p) => {
+            if (!legendData.includes(p.seriesName)) return false;
+            if (seen.has(p.seriesName)) return false;
+            seen.add(p.seriesName);
+            return true;
+          })
+          .map((p) => {
+            const v = (p.data as any[])[1] as number;
+            return `${p.marker} ${p.seriesName}: ${v.toFixed(1)} ${unit}`;
```

Comment:
`XX.0` on all values looks a bit strange, and some values might be more precise fractions where 1dp is not enough detail. Ideally the precision would be defined on the backend and we'd round the number to that, but I don't think that's currently supported. I suggest we just don't round these numbers for now and show them exactly as precise as the admins entered, what do you think?

<img width="173" alt="image" src="https://github.com/user-attachments/assets/b590c30d-d732-4e10-896e-0a967d930887" />

Reply:
Also, ideally the tooltip marker would be the same shape as on the graph. I remember that being tricky in ECharts though, so let's leave it as-is if it's too much of a pain

<img width="234" alt="image" src="https://github.com/user-attachments/assets/3aad3495-999f-4a4e-99e7-233f76b7dc2b" />